src/tiled/automapper.cpp

```diff
@@ -42,6 +42,11 @@
 
 using namespace Tiled;
 
+// Should i leave this function here?
+inline int wrap(int value, int bound) {
```

Comment:
The location is fine for now, but please define the function `static` and put the `{` on the next line.

Reply:
`inline static` or `static inline`? I know there's no difference, but maybe there is a convention in Tiled's code?

Reply:
Actually I doubt there is any precedence here since I don't really see the point of defining a function both `inline` and `static`.

I actually only use `inline` when I need to use it. That is, to avoid multiple-definition problems for function definitions in header files that are outside class definitions. In any other case it's at best a hint that the compiler is free to ignore (and probably will, it'll inline only when it thinks it's a good idea anyway).

The `static` however is appropriate here, since we don't intend the function to be used from another translation unit. Defining it as static makes it inaccessible outside of its translation unit.

Reply:
> Actually I doubt there is any precedence here since I don't really see the point of defining a function both `inline` and `static`.

Maybe I should search the code before raising doubts. So, there's precedence for both `inline static` and `static inline`, with the latter a clear winner. However, of 9 total occurrences, 8 were inherited from 3rd party code. The one remaining one, I'm probably going to change to just `static` soon. :P

Reply:
Not a problem, I will leave it just as `static`. Commiting the code again.